interface.py

```diff
@@ -75,52 +79,54 @@ def switch_power_down(self):
             else:
                 self.power_down = False
 
+
     def confirm_selection(self):
         """
-        When indicator is False the player can choose cards and switch PD
-        When is Truh the player ended the selection of cards
+        When indicator is False the player can choose cards and switch Power Down
+        When is True the player ended the selection of cards
         """
         self.indicator = True
 
-# cards_count, first_number, last_number
+
 def create_card_pack():
-    cards_move = {'back_up': [-1, 6, 250, 299],
-                    'move1': [1, 18, 300, 399],
-                    'move2': [2, 12, 400, 499],
-                    'move3': [3, 6, 500, 599],
-                    }
-    cards_rotation = {'u_turn': [Rotation.U_TURN, 6, 50, 99],
-                    'left': [Rotation.LEFT, 18, 100, 199],
-                    'right': [Rotation.RIGHT, 18, 200, 299],
-                    }
+    movement_cards = [(-1, 6, 250),
+                    (1, 18, 300),
+                    (2, 12, 400),
+                    (3, 6, 500),
+                    ]
+    rotation_cards = [(Rotation.U_TURN, 6, 50),
+                    (Rotation.LEFT, 18, 100),
+                    (Rotation.RIGHT, 18, 200),
+                    ]
     card_pack = []
 
-    for name, number in cards_move.items():
-        for i in range(number[1]):
-            for i in range(number[2], number[3], 5):
-                card_pack.append(MovementCard(i, number[0])) # [MovementCard(690, -1)...][]
-    for name, number in cards_rotation.items():
-        for i in range(number[1]):
-            for i in range(number[2], number[3], 5):
-                card_pack.append(RotationCard(i, number[0])) # [RotationCard(865, Rotation.LEFT)....]
-    shuffle(card_pack)
+    for movement, cards_count, first_number in movement_cards:
+        for i in range(cards_count):
```

Comment:
nebylo by lepší použít konkrétní jméno místo toho "i"?

Reply:
Je to provizorní balíček a stejně je blbě, takže asi ani ne.

Reply:
Ne, tak blbě to není, vlastně jsem to opravila, ale co ti vadí na i? :-D

Reply:
Osobně mi nevadí nic, ale když jsem si pouštěla nějaký minulý záznam ze srazu, tak o tom mluvil Petr, že by měly být konkrétní jména ? Ale třeba si to blbě pamatuju :-D Takže approve? :-D

Reply:
jako jo, pokud procházíš třeba seznam něčeho konkrétního for cyklem, ale toto je prostě range(int)
Ale nevím, jak to říkal Petr.